### ISSUE TITLE
Processed expression vectors were not created for GSE157902.2 on import of RNA-Seq data

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/Gemma#481]

**Comment by ppavlidis** [comment]
Just to keep things moving along I ran makeProcessedData on this data set. As to why it filed to happen on import still needs investigation via the logs.

But this experiment is unusual in that it was split by the curators.

**Comment by ppavlidis** [comment]
Still applies in 2.0-alpha; needs focused follow-up. Audited during 2.0-alpha bug triage. Root cause for the missing processed vectors after split-experiment import on GSE157902.2 was never identified; needs log review against the current importer.

**Comment by ppavlidis** [comment]
Re-walked the current code paths for the split-experiment flow during the 2.0-alpha bug audit and did **not** identify an obvious code defect — leaving open pending log review against a reproducer.

Findings (current state, `agent-rnaseq-qt-ingestion`):

1. `SplitExperimentServiceImpl.split` already calls `preprocessor.process(split)` for each split when `foundPreferred && postProcess`, so processed vectors should be created post-split.

2. Two paths that could silently skip processed-vector creation on GSE157902.2-shaped input:
   - **Non-DOUBLE QT**: line 130 throws `UnsupportedOperationException` for any QT whose representation isn't DOUBLE. For an RNA-seq EE imported with both an integer `Counts` QT and a DOUBLE `log2cpm` preferred QT, the first non-DOUBLE QT seen aborts the entire split. That would explain a clean import-but-no-vectors outcome if the operator caught the throw at the CLI layer.
   - **`foundPreferred=false`**: `foundPreferred` is only set to true inside the `findAndThaw(qt)` loop, gated on `!vectors.isEmpty()`. If the preferred QT's raw vectors were already detached at split time (because some earlier pipeline step removed them), the loop sets `foundPreferred=false` and `preprocessor.process` is silently skipped per the `if (foundPreferred && postProcess)` guard.

Next steps: needs a reproducer (or the original GSE157902.2 import log) to confirm which of the two branches was hit. Without that, any speculative fix risks changing semantics on the working majority of splits. Happy to wire this into the next 2.0-alpha test rerun if someone can grab a fresh log.

**Comment by ppavlidis** [comment]
Gemma 2 disposition: Won't fix
